### PR TITLE
Fix problems with id and name. Count is optional (assumed 1). Id/name is optional for replenishing.

### LINKED ISSUE #35: Account if user tries to add an item with matching name but wrong id number

### PR DESCRIPTION
**Pull request by bernarduskrishna**
Fix #35

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/AddCommandParser.java [hunk 1]
```diff
@@ -29,14 +29,20 @@ public AddCommand parse(String args) throws ParseException {
...
                 || !argMultimap.getPreamble().isEmpty()) {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddCommand.MESSAGE_USAGE));
         }
 
-        Name name = ParserUtil.parseName(argMultimap.getValue(PREFIX_NAME).get());
-        String id = ParserUtil.parseId(argMultimap.getValue(PREFIX_ID).get());
-        Integer count = ParserUtil.parseCount(argMultimap.getValue(PREFIX_COUNT).get());
+        Name name = arePrefixesPresent(argMultimap, PREFIX_NAME)
+                ? ParserUtil.parseName(argMultimap.getValue(PREFIX_NAME).get())
+                : new Name("dummy name");
```
Comment (wangpeialex) [inline src/main/java/seedu/address/logic/parser/AddCommandParser.java:39]: Can consider using `StringUtil.generateRandomString()` for dummy names

File: src/main/java/seedu/address/logic/parser/AddCommandParser.java [hunk 2]
```diff
@@ -29,14 +29,20 @@ public AddCommand parse(String args) throws ParseException {
...
+        String id = arePrefixesPresent(argMultimap, PREFIX_ID)
+                ? ParserUtil.parseId(argMultimap.getValue(PREFIX_ID).get())
+                : "999999";
```
Comment (wangpeialex) [inline src/main/java/seedu/address/logic/parser/AddCommandParser.java:42]: Can switch to java library `UUID.randomUUID().toString()` for dummy ids.

File: src/main/java/seedu/address/model/Model.java [hunk 3]
```diff
@@ -124,4 +124,11 @@
      * @return the Item with the same name.
      */
     Item getItemWithName(String name);
+
+    /**
+     * Gets Item with the same name.
+     * @param id The name of the item that wants to be searched
```
Comment (wangpeialex) [inline src/main/java/seedu/address/model/Model.java:130]: Should add a blank line between description and parameters

**Review by wangpeialex — CHANGES_REQUESTED** [review]
Just a few minor changes need to be done. LGTM.

File: src/main/java/seedu/address/logic/parser/AddCommandParser.java (hunk 2 above)
Comment (bernarduskrishna) [reply src/main/java/seedu/address/logic/parser/AddCommandParser.java:42]: Yup same here as above

**Pull request merged**